src/main/java/org/alliancegenome/curation_api/services/helpers/gff3/Gff3UniqueIdHelper.java

```diff
@@ -0,0 +1,39 @@
+package org.alliancegenome.curation_api.services.helpers.gff3;
+
+import java.util.Map;
+
+import org.alliancegenome.curation_api.enums.BackendBulkDataProvider;
+import org.alliancegenome.curation_api.model.ingest.dto.fms.Gff3DTO;
+import org.alliancegenome.curation_api.services.helpers.UniqueIdGeneratorHelper;
+
+import jakarta.enterprise.context.RequestScoped;
+
+@RequestScoped
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.design.HideUtilityClassConstructorCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Utility classes should not have a public or default constructor.